deployment/docker/REQUIREMENTS.txt

```diff
@@ -1,33 +1,82 @@
+alabaster==0.7.10
+amqp==1.4.9
+anyjson==0.3.3
+astroid==1.6.1
+Babel==2.5.3
+beautifulsoup4==4.6.0
+billiard==3.3.0.23
+cairocffi==0.8.0
+CairoSVG==2.1.3
+celery==3.1.25
+cffi==1.11.5
+coverage==4.5.1
+cssselect2==0.2.1
+defusedxml==0.5.0
 Django==1.8.14
-psycopg2
-pytz
-django-braces
-django-extra-views
-django-model-utils
+django-allauth>=0.33,<0.34
+django-betterforms==1.1.4
+django-bootstrap-form==3.3
+django-braces==1.12.0
+django-cors-headers==2.1.0
+django-crispy-forms==1.4.0
+django-devserver==0.8.0
+django-extra-views==0.10.0
+django-filter==1.1.0
+django-forms-bootstrap==3.1.0
+django-geojson==2.8.0
+django-grappelli==2.7.3
+django-haystack==2.5.1
+django-leaflet==0.16.0
+django-model-utils==3.1.1
+django-nested-inline==0.3.7
 django-pipeline==1.6.10
-nodeenv
-raven
-django-user-map
-django-forms-bootstrap
-# Django rest framework support
+django-user-map==1.1.1
 djangorestframework==3.5.3
-markdown
-django-filter
+djangorestframework-gis==0.12
+docutils==0.14
 elasticsearch==5.0.1
-django-leaflet==0.16.0
-django-geojson==2.8.0
-django-nested-inline
-django-grappelli==2.7.3
-django-crispy-forms==1.4.0
-setuptools==18.5
-WeasyPrint
-requests<2.12
-beautifulsoup4
-django-allauth==0.33.0
-django-betterforms
-celery<3.2
-more_itertools
-# Django haystack support
-git+https://github.com/dimasciput/django-haystack.git
-# Cors header
-django-cors-headers==2.1.0
\ No newline at end of file
+factory-boy==2.10.0
+Faker==0.8.11
+flake8==3.5.0
+html5lib==1.0.1
+imagesize==1.0.0
+isort==4.3.4
+Jinja2==2.10
+kombu==3.0.37
+lazy-object-proxy==1.3.1
+Markdown==2.6.11
+MarkupSafe==1.0
+mccabe==0.6.1
+more-itertools==4.1.0
+nodeenv==1.2.0
+oauthlib==2.0.6
+packaging==17.1
+pdfrw==0.4
+pep8==1.7.1
```

Comment:
@sonlinux, You don't need this in production, please put these in REQUIREMENTS-DEV.txt